.pre-commit-config.yaml

```diff
@@ -29,3 +29,7 @@ repos:
         args:
           - --in-place
           - --remove-all-unused-imports
+  - repo: https://github.com/RobertCraigie/pyright-python
+    rev: v1.1.392
```

Comment:
In PR description, please add a link towards a page that recommends (endorses) this extension.

Reply:
added